server/authorizer/authorizer.go

```diff
@@ -139,3 +205,29 @@ func parsePath(path string) ([]interface{}, error) {
 	}
 	return sl, nil
 }
+
+type authorizerCachedBody struct {
+	parsed interface{}
+}
+
+type authorizerCachedBodyKey string
+
+const ctxkey authorizerCachedBodyKey = "authorizerCachedBodyKey"
+
+// SetBodyOnContext adds the parsed input value to the context. This function is only
+// exposed for test purposes.
+func SetBodyOnContext(ctx context.Context, x interface{}) context.Context {
+	return context.WithValue(ctx, ctxkey, authorizerCachedBody{
+		parsed: x,
+	})
+}
+
+// GetBodyOnContext returns the parsed input from the request context if it exists.
+// The authorizer saves the parsed input on the context when it runs.
+func GetBodyOnContext(ctx context.Context) (interface{}, bool) {
+	input, ok := ctx.Value(ctxkey).(authorizerCachedBody)
```

Comment:
Do we need to check for a `nil` value here ?

Reply:
No, we should be fine because `nil.(authorizerCachedBody)` should just evaluate to `nil, false`